### PR TITLE
Add NUS Buses App

### PR DESCRIPTION
**Pull request by kohchihao**

### CODE REVIEW

File: apps.json [hunk 1]
```diff
@@ -1,4 +1,12 @@
 [
+{
+  "name": "NUS Buses",
+  "description": "Download the NUS Buses app to get real up to date bus arrival timings, bus routes and easily favourite the bus stops you prefer. It comes with a modern UI that is responsive and intuitive. Still in active development(2018)",
+  "author": "Marcus Koh Chi Hao",
+  "url": "https://goo.gl/jSDnQM",
+  "icon_url": "https://scontent.fsin3-1.fna.fbcdn.net/v/t1.0-1/p480x480/26113864_141658423281922_7246164886583028167_n.png?oh=54f8df88a59e21963907eb30d2d1319e&oe=5AFCD837",
+  "tags": ["nus", "buses"]
```
Comment (taneliang) [inline apps.json]: Probably shouldn't have an "nus" tag since this is an NUS-only site after all haha

File: apps.json [hunk 2]
```diff
@@ -1,4 +1,12 @@
 [
+{
+  "name": "NUS Buses",
+  "description": "Download the NUS Buses app to get real up to date bus arrival timings, bus routes and easily favourite the bus stops you prefer. It comes with a modern UI that is responsive and intuitive. Still in active development(2018)",
+  "author": "Marcus Koh Chi Hao",
+  "url": "https://goo.gl/jSDnQM",
```
Comment (taneliang) [inline apps.json]: Are you sure you only want to link to your iOS app? You could link to a website that has links to both your Android and iOS versions, but if you don't have one at the moment you can always submit another PR in the future :)

File: apps.json [hunk 3]
```diff
@@ -1,4 +1,12 @@
 [
+{
+  "name": "NUS Buses",
+  "description": "Download the NUS Buses app to get real up to date bus arrival timings, bus routes and easily favourite the bus stops you prefer. It comes with a modern UI that is responsive and intuitive. Still in active development(2018)",
```
Comment (taneliang) [inline apps.json]: /nit please add a space before the bracket and a period at the end

**Review by taneliang — CHANGES_REQUESTED** [review]

File: apps.json (hunk 2 above)
Comment (kohchihao) [reply apps.json]: yeah ios currently. dont have a website to do that yet.

**Review by taneliang — APPROVED** [review]

**Pull request merged**